src/index.js

```diff
@@ -124,7 +124,7 @@ import {
   rgbBrightness,
 } from './utils';
 
-import { version } from '../package.json';
+const version = import.meta.env.version;
```

Comment:
https://github.com/snowpackjs/snowpack/discussions/1262

Reply:
Is this `import.meta.env` from snowpack? This looks a bit mystic to me and I can't say I like having build-tools specific imports in the main source code.

Reply:
I'm not at computer now but just had an idea of how to avoid this and keep imports the same!